volatility3/framework/plugins/linux/malware/malfind.py

```diff
@@ -51,14 +61,41 @@ def _list_injections(
 
         proc_layer = self.context.layers[proc_layer_name]
 
+        # Allowing a dump_size of 0 (no dump)
+        dump_size = (
+            self.config.get("dump-size")
+            if self.config.get("dump-size") is not None
+            else 64
+        )
```

Comment:
```suggestion
        dump_size = self.config.get("dump-size", None) or 64
```
This will prevent you setting a dump-size of 0, but that's actually helpful and it's a lot easier to read than the previous 4-liner...  5;)